.evergreen/config.yml

```diff
@@ -424,28 +422,53 @@ tasks:
     - func: "attach node xunit results"
 
 - name: publish-snapshot
+  commands:
+    - func: "publish snapshot"
+
+- name: upload-java
   depends_on:
-    - variant: ubuntu2004-64
-      name: prep-c-driver-source
-    - variant: rhel-62-64-bit
-      name: build-and-test-java
     - variant: rhel-62-64-bit
       name: build-and-test-and-upload
     - variant: rhel-67-s390x
       name: build-and-test-and-upload
-    - variant: ubuntu1604-arm64
-      name: build-and-test-java
     - variant: ubuntu1604-arm64
       name: build-and-test-and-upload
-    - variant: macos_x86_64
-      name: build-and-test-java
+    - variant: windows-test
+      name: build-and-test-and-upload
     - variant: macos
       name: build-and-test-and-upload
-    - variant: windows-test
-      name: build-and-test-and-upload # Todo update once java build passing on windows
+    - variant: macos_x86_64
+      name: build-and-test-and-upload
   commands:
-    - func: "fetch source"
-    - func: "publish snapshot"
+    - command: shell.exec
+      params:
+        script: mkdir all
+    - func: "download tarball"
+      vars: { variant_name: "rhel-62-64-bit" }
+    - func: "download tarball"
+      vars: { variant_name: "rhel-67-s390x" }
+    - func: "download tarball"
+      vars: { variant_name: "ubuntu1604-arm64" }
+    - func: "download tarball"
+      vars: { variant_name: "windows-test" }
+    - func: "download tarball"
+      vars: { variant_name: "macos" }
+    - func: "download tarball"
+      vars: { variant_name: "macos_x86_64" }
```

Comment:
> RHEL 7.2 on zSeries - cannot find the artefact.
> RHEL 7.1 ppc64el - No ppc64el library included - should this be added?

Is this resolved by including `rhel72-zseries-test` and `rhel-71-ppc64el` be included in the libmongocrypt-java.tar.gz and adding to the `jnaMapping` in build.gradle.kts?